src/Orchard.Web/Modules/Orchard.MediaLibrary/Scripts/media-library.js

```diff
@@ -71,8 +71,11 @@ $(function () {
 
         var listWidth = $('#media-library-main-list').width();
         var listHeight = $('#media-library-main-list').height();
-        var itemWidth = $('.thumbnail').first().width();
-        var itemHeight = $('.thumbnail').first().height();
+        var itemWidth = $('#media-library-main-list li').first().width();
+        var itemHeight = $('#media-library-main-list li').first().height();

```

Comment:
On load, the .thumbnail class has 0 height. This is what was causing the infinity value for count.
Changing this to the containing li item allows for a set size based on the CSS.